exovetter/vetters.py

```diff
@@ -84,31 +84,47 @@ def plot(self, tce, lightcurve):
 class ModShift(BaseVetter):
     """Modshift vetter."""
 
-    def __init__(self, **kwargs):
+    def __init__(self, lc_name="flux", **kwargs):
         self.metrics = None
+        self.lc_name = lc_name
 
     def run(self, tce, lightcurve):
-        """What I want this to look like, but it doesn't work yet
         """
-        time = lightcurve.time
-        flux = lightcurve.flux
-        time_offset_str = lightcurve.time_format
+        run ModShift vetter
+
+        Parameters
+        ---------
+        tce : tce object
+        tce object is a dictionary that contains information about the tce
+        to vet, like period, epoch, duration, depth
+
+        lc : lightkurve object
+        lightkurve object with the time and flux to use for vetting.
+
+        Runs modshift.compute_modeshift_metrics to populate the vetter object.
```

Comment:
I can't tell if this is part of `lc` description or what. If this is a general statement, please move it before Parameters.